docker/intel-dgpu-driver.Dockerfile

```diff
@@ -1,42 +1,51 @@
 # Copyright (c) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+ARG DIR=/build
```

Comment:
@uMartinXu The DIR arg is not for users to set (although technically they can) but the default value is `/build`. Since it is the variable used for `WORKDIR`, we need it in both the stages of the build, hence we need to declare it before the first `FROM` statement and again define it in the following build stages to use.

Reply:
Will we change the DIR other than /build quite often in the future? If not, we don't have to define a new ARG, Right?

Reply:
Hi @uMartinXu yes thats correct, its not good to let users make interfaces for safety.. Better to keep less interfaces for more safety. I updated the PR. Please review.